Index: .idea/tv3.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/tv3.iml b/.idea/holdplotter.iml
rename from .idea/tv3.iml
rename to .idea/holdplotter.iml
--- a/.idea/tv3.iml	(revision d300d5070e7909476ee9884ace16d9468665c07d)
+++ b/.idea/holdplotter.iml	(date 1729277686113)
@@ -2,7 +2,7 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Python 3.12" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="pythonProject2" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
-</module>
+</module>
\ No newline at end of file
Index: DataClass2.py
===================================================================
diff --git a/DataClass2.py b/DataClass2.py
deleted file mode 100644
--- a/DataClass2.py	(revision d300d5070e7909476ee9884ace16d9468665c07d)
+++ /dev/null	(revision d300d5070e7909476ee9884ace16d9468665c07d)
@@ -1,84 +0,0 @@
-from DataClass import  TestCandle, HoldCandle , Untested
-from PlotClass import UntestedPlot
-from tvDatafeed import Interval
-
-from DataClass import intervals
-
-INTERVALS = ['in_1_minute', 'in_3_minute', 'in_5_minute', 'in_15_minute', 'in_1_hour', 'in_4_hour', 'in_daily']
-COLORS = ['grey', 'pink', 'green', 'orange', 'blue', 'yellow', 'red']
-N_BARS = [5000, 1666, 1000, 333, 83, 21, 3]
-
-INTERVALS = ['in_1_minute', 'in_3_minute', 'in_5_minute', 'in_15_minute', 'in_1_hour', 'in_4_hour', 'in_daily']
-COLORS = ['grey', 'pink', 'green', 'orange', 'blue', 'yellow', 'red']
-N_BARS = [5000, 1666, 1000, 333, 83, 21, 3]
-colors_dict = dict(zip(INTERVALS, COLORS))
-n_dict = dict(zip(INTERVALS, N_BARS))
-TC = TestCandle(Interval.in_1_minute, n_bars=5000)
-
-interval = intervals[5]
-
-n_bars=n_dict[interval]
-color = colors_dict[interval]
-
-
-
-holdcandle = HoldCandle(interval, n_bars)
-
-class TimeFrame(HoldCandle):
-    def __init__(self, holdcandle):
-
-        self.candle = holdcandle
-        self.untapped = None
-        self.dict = None
-        self.df = None
-        self.utp = None
-        self.fig = None
-
-
-        def set_untested(self):
-
-            self.untapped = Untested(self.candle, TC)
-
-            self.dict = self.untested.__dict__
-            self.df = self.dict['holdcandle'].df
-            self.utp = UntestedPlot(self.df, **self.dict)
-            self.fig = self.utp.plot()
-            self.fig.show()
-
-
-
-class ThreeDay:
-    def __init__(self, interval, n_bars, color):
-
-
-
-        self.untested = None
-        self.dict =None
-        self.df =None
-        self.utp =None
-        self.fig = None
-
-        self.set_untested()
-
-
-        def set_untested(self):
-
-            self.untested = Untested(self.candle, TC)
-            self.dict = self.untested.__dict__
-            self.df = self.dict['holdcandle'].df
-            self.utp = UntestedPlot(self.df, **self.dict)
-            self.fig = self.utp.plot()
-            self.fig.show()
-
-
-# print('dsffd')
-#
-# for i in INTERVALS:
-#     print(i)
-#
-#     tf = ThreeDay(i, n_dict[i], n_dict[i], colors_dict[i])
-
-# @classmethod
-    # def get_testcandle(interval=Interval.in_1_minute, nbars=5000):
-
-print('dfghdf')
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectModuleManager\">\r\n    <modules>\r\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/tv3.iml\" filepath=\"$PROJECT_DIR$/.idea/tv3.iml\" />\r\n    </modules>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
--- a/.idea/modules.xml	(revision d300d5070e7909476ee9884ace16d9468665c07d)
+++ b/.idea/modules.xml	(date 1729278207031)
@@ -2,7 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/tv3.iml" filepath="$PROJECT_DIR$/.idea/tv3.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/holdplotter.iml" filepath="$PROJECT_DIR$/.idea/holdplotter.iml" />
     </modules>
   </component>
 </project>
\ No newline at end of file
